### PR TITLE
Siemens_Vision600_ZrNEMAIQ

### PR DESCRIPTION
**Pull request by KrisThielemans**
mostly just adding Siemens_Vision600_ZrNEMAIQ, but had to change vmax from OSEM_mage.max() to the 99th percentile to get anything.

### CODE REVIEW

File: SIRF_data_preparation/data_QC.py [hunk 1]
```diff
@@ -73,10 +73,10 @@ def plot_image(image, save_name=None, transverse_slice=-1, coronal_slice=-1, sag
         coronal_slice = image.dimensions()[1] // 2
     if sagittal_slice < 0:
         sagittal_slice = image.dimensions()[2] // 2
+    arr = image.as_array()
     if vmax is None:
-        vmax = image.max()
+        vmax = np.percentile(arr, 99) / .99
```
Comment (casperdcl) [inline SIRF_data_preparation/data_QC.py:78]: wait why `/.99`?

```suggestion
        vmax = np.percentile(arr, 99)
```

or maybe you meant:

```suggestion
        vmax = np.percentile(arr, 99.995)
```

File: SIRF_data_preparation/plot_BSREM_metrics.py [hunk 2]
```diff
@@ -38,7 +41,7 @@
 settings = get_settings(scanID)
 slices = settings.slices
 
-cmax = OSEM_image.max()
+cmax = numpy.percentile(OSEM_image.as_array(), 99) / .99
```
Comment (casperdcl) [inline SIRF_data_preparation/plot_BSREM_metrics.py:44]: same question here

File: petric.py [hunk 3]
```diff
@@ -316,7 +317,10 @@ def get_image(fname):
         (SRCDIR / "Siemens_Vision600_Hoffman", OUTDIR / "Vision600_Hoffman",
          [MetricsWithTimeout(outdir=OUTDIR / "Vision600_Hoffman", **DATA_SLICES['Siemens_Vision600_Hoffman'])]),
         (SRCDIR / "NeuroLF_Esser_Dataset", OUTDIR / "NeuroLF_Esser",
-         [MetricsWithTimeout(outdir=OUTDIR / "NeuroLF_Esser", **DATA_SLICES['NeuroLF_Esser_Dataset'])])]
+         [MetricsWithTimeout(outdir=OUTDIR / "NeuroLF_Esser", **DATA_SLICES['NeuroLF_Esser_Dataset'])]),
+        (SRCDIR / "Siemens_Vision600_ZrNEMAIQ", OUTDIR / "Vision600_ZrNEMAIQ", [
+            MetricsWithTimeout(outdir=OUTDIR / "Siemens_Vision600_ZrNEMAIQ",
+                               **DATA_SLICES['Siemens_Vision600_ZrNEMAIQ'])])]
```
Comment (casperdcl) [inline petric.py:323]:
```suggestion
            MetricsWithTimeout(outdir=OUTDIR / "Vision600_ZrNEMAIQ", **DATA_SLICES['Siemens_Vision600_ZrNEMAIQ'])])]
```

File: SIRF_data_preparation/run_BSREM.py [hunk 4]
```diff
@@ -50,25 +55,32 @@
 # log.info("Finding files in %s", srcdir)
 
 settings = get_settings(scanID)
+if num_subsets is None:
+    num_subsets = settings.num_subsets
+else:
+    num_subsets = int(num_subsets)
```
Comment (casperdcl) [inline SIRF_data_preparation/run_BSREM.py:61]:
```suggestion
num_subsets = settings.num_subsets if num_subsets is None else int(num_subsets)
```

File: SIRF_data_preparation/data_QC.py (hunk 1 above)
Comment (KrisThielemans) [reply SIRF_data_preparation/data_QC.py:78]: I divided by .99 just to get it a bit closer to the original max (only works if linear distribution of course).

File: SIRF_data_preparation/plot_BSREM_metrics.py (hunk 2 above)
Comment (KrisThielemans) [reply SIRF_data_preparation/plot_BSREM_metrics.py:44]:
```suggestion
cmax = numpy.percentile(OSEM_image.as_array(), 99.995)
```

**Review by casperdcl — APPROVED** [review]

**Pull request merged**